features/step_definitions/web_steps.rb

```diff
@@ -197,6 +197,10 @@ def fill_in
   end
 end
 
+Then "the current page is {}" do |page_name|
+  assert_current_path path_to(page_name)
+end
```

Comment:
This looks familiar...

https://github.com/3scale/porta/pull/3526/files#diff-485d7e9c4e7cafba9be795aaae354ff30ac48a9ad9b8a6bf8420e72596bc4995R200-R202

Reply:
🍒⛏️